polygon-daemon/templates/configmap.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.configMap }}
+data:
+  entrypoint.sh: |
+    #!/bin/sh
+
+    # exit script on any error
+    set -e
+
+    # Set Heimdall Home Directory
+    HEIMDALLD_HOME=/heimdall-home
```

Comment:
Think it might make sense to try and structure the values to reference the same path here as the PVC mount?